### ISSUE TITLE
Add additional an option to include navigation keys

### ISSUE THREAD

**Issue by akitchen** [keycastr/keycastr#296]
This is covered across a few other issues, but it would be good to have an option to add navigation keys to the current display option. It could compose with "All Modified Keys" or "Command Keys Only".

I think this would include esc, tab/shift-tab, delete, arrows, numeric pad keys, spacebar, and return.

**Comment by AbdelrahmanHafez** [comment]
Are you open for PRs for this? We can start with numpad keys (my main use-case here), and once we establish an accepted design we can tackle the rest.

**Comment by akitchen** [comment]
I appreciate the offer, but in this case - an update is coming soon which will launch the new minimal visualizer which will support this use case. And then the plan is to unify settings across the different visualizers as much as we can (it makes more sense to do that when there are three or more). Development has been slow but this is almost ready!

**Comment by AbdelrahmanHafez** [comment]
Makes sense, I created the PR anyway and closed it immediately since I had prepared it yesterday already. Hopefully it'll help somehow with the new implementation.

Thanks for maintaining KeyCastr!
